app/views/shared/forms/_applicant_form.html.erb

```diff
@@ -18,7 +18,14 @@
       <%= form.govuk_text_field :first_name, label: { text: "First name" }, width: "three-quarters" %>
       <%= form.govuk_text_field :last_name, label: { text: "Last name" }, width: "three-quarters" %>
     <% end %>
-
+    <%= form.govuk_radio_buttons_fieldset :changed_last_name,
+                                          legend: { size: "m", tag: "h2", text: t(".changed_last_name") },
```

Comment:
Should this be an h2? It doesn't like it from the designs but happy to be corrected

Reply:
👍 have removed